docs/migration-guide/overview.mdx

```diff
@@ -1104,6 +1104,17 @@ plugins: [
 
 If you have custom features for `@payloadcms/richtext-lexical` you will need to migrate your code to the new API. Read more about the new API in the [documentation](https://payloadcms.com/docs/rich-text/building-custom-features).
 
+## Reserved Field names
+
+Payload reserves certain field names for internal use. If your v2 collection uses any of these names, the field will be sanitized from the config, causing errors when deploying your application. Make sure to rename any fields that conflict with the reserved names:
+
+- `__v`
+- `salt`
+- `hash`
+- `file`
```

Comment:
For upload enabled collections you're missing the following:
- url
- thumbnailURL
- filename
- mimeType

Condition upon imageSizes in config
- sizes
- filesize
- width
- height
- focalX
- focalY